Geometry/MTDCommonData/data/mtdMaterial/v3/mtdMaterial.xml

```diff
@@ -154,6 +154,11 @@
         <rMaterial name="materials:Air"/>
       </MaterialFraction>
     </CompositeMaterial>
+    <CompositeMaterial name="Borated_Polyethyl." density="950*mg/cm3" symbol=" " method="mixture by weight">
```

Comment:
Why is this seemingly duplicate material introduced? It has the same density as `materials:Borated Polyethyl.`.

Reply:
@cvuosalo this is essentially a workaround to let  the material budget validation code parsing without issues the tables with the material names, it is unclear to me how to handle blanks in the middle (it was a bad idea to introduce them in first place, I would say). I mimicked what done already for other similar situations, see for instance 

https://cmssdt.cern.ch/lxr/source/Geometry/CMSCommonData/data/materials.xml#1157

If you have a better solution to suggest I will be happy to consider it, I do not like this solution either, but "it just works".

Reply:
@fabiocos Thanks, I understand now. It is unfortunate that we are stuck with this name containing a space, but I see it is widely used, so it would be some effort to change.